lib/src/emoji_picker.dart

```diff
@@ -178,30 +180,35 @@ class _EmojiPickerState extends State<EmojiPicker> {
   Future<void> _updateEmojis() async {
     categoryEmoji.clear();
     if (widget.config.showRecentsTab) {
-      recentEmoji = await _getRecentEmojis();
+      recentEmoji = await EmojiPickerUtils.getRecentEmojis();
       final recentEmojiMap = recentEmoji.map((e) => e.emoji).toList();
       categoryEmoji.add(CategoryEmoji(Category.RECENT, recentEmojiMap));
     }
-    categoryEmoji.addAll([
-      CategoryEmoji(Category.SMILEYS,
-          await _getAvailableEmojis(emoji_list.smileys, title: 'smileys')),
-      CategoryEmoji(Category.ANIMALS,
-          await _getAvailableEmojis(emoji_list.animals, title: 'animals')),
-      CategoryEmoji(Category.FOODS,
-          await _getAvailableEmojis(emoji_list.foods, title: 'foods')),
-      CategoryEmoji(
-          Category.ACTIVITIES,
-          await _getAvailableEmojis(emoji_list.activities,
-              title: 'activities')),
-      CategoryEmoji(Category.TRAVEL,
-          await _getAvailableEmojis(emoji_list.travel, title: 'travel')),
-      CategoryEmoji(Category.OBJECTS,
-          await _getAvailableEmojis(emoji_list.objects, title: 'objects')),
-      CategoryEmoji(Category.SYMBOLS,
-          await _getAvailableEmojis(emoji_list.symbols, title: 'symbols')),
-      CategoryEmoji(Category.FLAGS,
-          await _getAvailableEmojis(emoji_list.flags, title: 'flags'))
-    ]);
+
+    Map.fromIterables([
+      Category.ACTIVITIES,
+      Category.ANIMALS,
+      Category.FLAGS,
+      Category.FOODS,
+      Category.OBJECTS,
+      Category.SMILEYS,
+      Category.SYMBOLS,
+      Category.TRAVEL
```

Comment:
Correct order is important here, the order of the categories displayed in the widget is decided here 😅

Reply:
I'm so sorry, didn't notice.